server_admin/topics/users/create-user.adoc

```diff
@@ -1,13 +1,10 @@
-[creat-new-users]
+[id="proc-creating-new-users_{context}"]
```

Comment:
This should be "Creating a user"
Generally, I think singular is best for procedures.  Also, "new" and "create" are redundant.